oap-ape/ape-java/ape-common/src/main/java/com/intel/ape/util/ZStdUtils.java

```diff
@@ -0,0 +1,45 @@
+package com.intel.ape.util;
```

Comment:
add license please. And I noticed that zstd-jni is using BSD license, any risk? @winningsix  
https://github.com/luben/zstd-jni#license